### PR TITLE
Obr v2 5 1

### PR DESCRIPTION
**Pull request by Zhauner**

### CODE REVIEW

**Comment by Zhauner** [comment]
Tt

File: hl7conv/schemas/versions/v2_5_1/enums.py [hunk 1]
```diff
@@ -2874,8 +2874,8 @@ class MilitaryRankorGrade(str, Enum):
      E8 = "E8"
      E1 = "E1"
      E7 = "E7"
-     O1_____O9 = "O1 ... O9"
-     W1_____W4 = "W1 ... W4"
+     O1_O9 = "O1 ... O9"
```
Comment (IlyaKalosha) [inline hl7conv/schemas/versions/v2_5_1/enums.py:2877]: delete from code

File: hl7conv/schemas/versions/v2_5_1/enums.py [hunk 2]
```diff
@@ -2874,8 +2874,8 @@ class MilitaryRankorGrade(str, Enum):
...
+     W1_W4 = "W1 ... W4"
```
Comment (IlyaKalosha) [inline hl7conv/schemas/versions/v2_5_1/enums.py:2878]: delete from code

File: hl7conv/schemas/versions/v2_5_1/enums.py [hunk 3]
```diff
@@ -2379,11 +2379,11 @@ class DischargeDisposition(str, Enum):
      _07 = "07"
      _08 = "08"
      _09 = "09"
-     _10_____19 = "10 ... 19"
+     _10_19 = "10 ... 19"
```
Comment (IlyaKalosha) [inline hl7conv/schemas/versions/v2_5_1/enums.py:2382]: add each one separately and delete from code

**Review by IlyaKalosha — CHANGES_REQUESTED** [review]

**Review by IlyaKalosha — APPROVED** [review]

**Review by IlyaKalosha — APPROVED** [review]

**Comment by IlyaKalosha** [comment]
- OBX v2_5_1 validation 
- enums v2_5_1
- validators v2_5_1
- complex structs v2_5_1
- GH actions with linters

**Pull request merged**